src/screens/dashboard/DashboardScreen.tsx

```diff
@@ -28,12 +28,21 @@ export class DashboardScreen extends React.PureComponent<NavigationInjectedProps
   render() {
     const student = this.props.navigation.getParam('student');
 
+    const navigateToCreatePlan = () =>
+      this.props.navigation.navigate('PlanActivity', {
+        student,
+      });
+
     return (
       <View style={styles.container}>
         <FullScreenTemplate padded darkBackground>
           {student && <StudentPlanList student={student} />}
         </FullScreenTemplate>
-        <CreateStudentButton />
+        {student ? (
+          <CreateButton actionsName="create plan" onPress={navigateToCreatePlan} />
+        ) : (
+          <CreateButton actionsName="create student" onPress={Student.create} />
```

Comment:
This use case is invalid. Ignore for now scenario when there is no user. You can still add him from the users modal. There will be additional UX flow for the very first app launch.

Reply:
fixed